modules/tensor_mechanics/src/materials/ComputeFiniteStrainElasticStress.C

```diff
@@ -25,8 +25,8 @@ ComputeFiniteStrainElasticStress::ComputeFiniteStrainElasticStress(
     GuaranteeConsumer(this),
     _elasticity_tensor_name(_base_name + "elasticity_tensor"),
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_elasticity_tensor_name)),
-    _rotation_total(declareProperty<RankTwoTensor>("rotation_total")),
-    _rotation_total_old(getMaterialPropertyOldByName<RankTwoTensor>("rotation_total")),
+    _rotation_total(declareProperty<RankTwoTensor>(_base_name + "rotation_total")),
+    _rotation_total_old(getMaterialPropertyOldByName<RankTwoTensor>(_base_name + "rotation_total")),
```

Comment:
Necessary addition of `_base_name`

Reply:
Could you also add it to `ADComputeFinitStrainElasticStress`?

Reply:
Oh I forgot that regular and AD are still in separate files here. I just did. Thanks for catching that.